spec/yelp/endpoint/business_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:api_keys) { real_api_keys }
   let(:business) { 'yelp-san-francisco' }
   let(:client) { Yelp::Client.new(api_keys) }
+  let(:locale) { Hash[lang: 'fr'] }
```

Comment:
The API doesn't officially support a locale but we do call that group of params "locale parameters" so I followed in pattern